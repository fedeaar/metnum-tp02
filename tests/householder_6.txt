7.601648073625430868e+01 -5.167814750849462069e-01 -3.456737982345972071e+00 -3.675444839937409003e-01 -2.647109113200816655e+00 -9.936236161010089774e+00 -1.632853383697244709e+01 4.435881717393391099e+00 1.858759441813297331e+00 -4.768902240710369611e+00
-5.167814750849460959e-01 3.920266850774275724e+01 1.554478560461958869e+00 2.762090183579199998e-01 3.120065789167937087e+00 3.174163807016917166e-02 -2.316506416364997545e+00 1.025467793204225764e+01 3.799432562522612855e+00 1.798300295397355919e+00
-3.456737982345972071e+00 1.554478560461958647e+00 5.791739963183875517e+01 7.942939061702121439e-02 1.471286222859780723e+00 -3.765511338929256269e+00 -8.071628364973502556e+00 9.847183149750325271e+00 3.730565928914944873e+00 -3.893063934527079550e-01
-3.675444839937409003e-01 2.762090183579199998e-01 7.942939061702100623e-02 5.202332745299653993e+01 3.197129109903295574e-01 -3.668948028258456340e-01 -9.207133043416504403e-01 1.541475511389670761e+00 5.791675703345084969e-01 6.312686287845180322e-02
-2.647109113200820207e+00 3.120065789167937087e+00 1.471286222859780501e+00 3.197129109903297794e-01 5.096703224134429888e+01 -2.301132159295728563e+00 -7.268041588972077527e+00 1.614234701885786905e+01 6.031714537834070633e+00 1.520121122320191587e+00
-9.936236161010086221e+00 3.174163807016869981e-02 -3.765511338929256269e+00 -3.668948028258456895e-01 -2.301132159295729451e+00 6.583713643482900579e+01 -2.070255296904713660e+01 8.529355296180490598e+00 3.423860559546178184e+00 -5.299393508917679796e+00
-1.632853383697245064e+01 -2.316506416364996657e+00 -8.071628364973502556e+00 -9.207133043416504403e-01 -7.268041588972077527e+00 -2.070255296904714370e+01 6.031303845742967695e+01 3.458187600581408283e+00 1.729343212004516239e+00 -1.094055288176982721e+01
4.435881717393386658e+00 1.025467793204225764e+01 9.847183149750321718e+00 1.541475511389670761e+00 1.614234701885786549e+01 8.529355296180490598e+00 3.458187600581410948e+00 5.696574004807712299e+01 1.536691792723464545e+01 1.204171098036133181e+01
1.858759441813298663e+00 3.799432562522612855e+00 3.730565928914944873e+00 5.791675703345084969e-01 6.031714537834069745e+00 3.423860559546178628e+00 1.729343212004515129e+00 1.536691792723464900e+01 1.762052011391978823e+01 4.576981579249514098e+00
-4.768902240710371387e+00 1.798300295397356141e+00 -3.893063934527080661e-01 6.312686287845177546e-02 1.520121122320191365e+00 -5.299393508917680684e+00 -1.094055288176982543e+01 1.204171098036133181e+01 4.576981579249514098e+00 5.913665637556780297e+01
