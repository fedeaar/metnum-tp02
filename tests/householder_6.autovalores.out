1.000000000000000000e+01
1.000000000000000000e+04
1.000000000000000036e-10
9.300000000000002842e+01
8.400000000000000000e+01
7.799999999999998579e+01
6.000000000000000711e+01
5.800000000000000000e+01
5.199999999999999289e+01
4.700000000000001421e+01
3.700000000000002132e+01
1.500000000000001066e+01
1.200000000000000178e+01
